api/tests/test_asset_incident_report_api.py

```diff
@@ -141,3 +141,42 @@ def test_cant_allow_delete_incident_report(self, mock_verify_id_token):
         )
         self.assertEqual(response.data, {'detail': 'Method "DELETE" not allowed.'})
         self.assertEqual(response.status_code, 405)
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_can_allow_patch_incident_report_status(self, mock_verify_id_token):
+        mock_verify_id_token.return_value = {'email': self.admin_user.email}
+        data = {
+            "asset_incident_report": self.report_status.asset_incident_report.id,
+            "incident_report_state": "internal assessment",
+            "asset_state": "damaged",
+        }
+        response = client.put(
```

Comment:
You test method says you could `patch` a report but you're making a `put` request here.

Reply:
I've updated the tests to work with client.patch, @kmazi.